package.json

```diff
@@ -99,6 +99,6 @@
   },
   "peerDependencies": {
     "typescript": "*",
-    "webpack": "*"
+    "webpack": "^5.20.0"
```

Comment:
Thanks!  Why specifically 5.20.0 as the minimum?

Reply:
Because it's the version in the `devDependencies` section, so I assume the plugin is tested against this specific version.

Reply:
It is - but it's not expressing a minimum version for users. To my knowledge, anything 5+ should be fine